Formula/cockroach.rb

```diff
@@ -1,10 +1,10 @@
 class Cockroach < Formula
   desc "Distributed SQL database"
   homepage "https://www.cockroachlabs.com"
-  url "https://binaries.cockroachdb.com/cockroach-v20.2.1.darwin-10.9-amd64.tgz"
+  url "https://binaries.cockroachdb.com/cockroach-v20.2.2.darwin-10.9-amd64.tgz"
   version "20.2.1"
```

Comment:
Only saw this after merge, but I think the `version` field also needs updating here?